scripts/fixtures/users_collection_sample.json

```diff
@@ -0,0 +1 @@
+[{"name":"Suga", "role": "Prime Minister", "id": "5xeZHoQzgl7fEcVVzyIN"}]
```

Comment:
Lol let's add Kono Taro-san too 😆

Reply:
I thought about that =). That'd be fun ne.

Reply:
I did it. Let's be cheeky.